diff --git a/src/Controller/ProductController.php b/src/Controller/ProductController.php
index c528a80..cbaea7c 100644
--- a/src/Controller/ProductController.php
+++ b/src/Controller/ProductController.php
@@ -20,4 +20,14 @@ class ProductController extends AbstractController
             'products' => $products,
         ]);
     }
+
+    /**
+     * @Route("/api/products")
+     */
+    public function getProductsApi(ProductRepository $productRepository)
+    {
+        $products = $productRepository->findAll();
+
+        return $this->json(['data' => $products]);
+    }
 }
