diff --git a/src/Security/UserProvider.php b/src/Security/UserProvider.php
index ccccbf1..4416c7a 100644
--- a/src/Security/UserProvider.php
+++ b/src/Security/UserProvider.php
@@ -55,9 +55,7 @@ class UserProvider implements UserProviderInterface, PasswordUpgraderInterface
             throw new UnsupportedUserException(sprintf('Invalid user class "%s".', get_class($user)));
         }
 
-        // Return a User object after making sure its data is "fresh".
-        // Or throw a UsernameNotFoundException if the user no longer exists.
-        throw new \Exception('TODO: fill in refreshUser() inside '.__FILE__);
+        return $user;
     }
 
     /**
