diff --git a/src/Security/SimpleSSOAuthenticator.php b/src/Security/SimpleSSOAuthenticator.php
index 1d904f5..586f8b0 100644
--- a/src/Security/SimpleSSOAuthenticator.php
+++ b/src/Security/SimpleSSOAuthenticator.php
@@ -18,12 +18,15 @@ class SimpleSSOAuthenticator extends AbstractGuardAuthenticator
 
     public function getCredentials(Request $request)
     {
-        dd('credentials');
+        return [
+            'email' => $request->query->get('email'),
+            'name' => $request->query->get('name'),
+        ];
     }
 
     public function getUser($credentials, UserProviderInterface $userProvider)
     {
-        // todo
+        dd($credentials);
     }
 
     public function checkCredentials($credentials, UserInterface $user)
