diff --git a/src/Security/SimpleSSOAuthenticator.php b/src/Security/SimpleSSOAuthenticator.php
index 832b7d2..b3fb792 100644
--- a/src/Security/SimpleSSOAuthenticator.php
+++ b/src/Security/SimpleSSOAuthenticator.php
@@ -35,7 +35,7 @@ class SimpleSSOAuthenticator extends AbstractGuardAuthenticator
 
     public function checkCredentials($credentials, UserInterface $user)
     {
-        dd('check credentials');
+        return true;
     }
 
     public function onAuthenticationFailure(Request $request, AuthenticationException $exception)
