diff --git a/src/Controller/ProductController.php b/src/Controller/ProductController.php
index 9b507f6..7703272 100644
--- a/src/Controller/ProductController.php
+++ b/src/Controller/ProductController.php
@@ -12,7 +12,8 @@ class ProductController extends AbstractController
      */
     public function list()
     {
-        $pdo = new \PDO('sqlite:///Users/weaverryan/Sites/symfony_training/var/data.db');
+        $projectDir = __DIR__.'/../../';
+        $pdo = new \PDO('sqlite://'.$projectDir.'/var/data.db');
         $pdo->setAttribute(\PDO::ATTR_ERRMODE, \PDO::ERRMODE_EXCEPTION);
 
         $results = $pdo->query('SELECT * FROM product')->fetchAll();
