diff --git a/config/services.yaml b/config/services.yaml
index e5c518b..c5c8042 100644
--- a/config/services.yaml
+++ b/config/services.yaml
@@ -23,8 +23,9 @@ services:
 
     App\Controller\LuckyController:
         arguments:
-            $globalMinNumber: 5
             $logger: '@logger'
             $luckyNumberGenerator: '@App\Service\LuckyNumberGenerator'
 
     App\Service\LuckyNumberGenerator:
+        arguments:
+            $globalMinNumber: 5
diff --git a/src/Controller/LuckyController.php b/src/Controller/LuckyController.php
index 7e6d3b4..436faff 100644
--- a/src/Controller/LuckyController.php
+++ b/src/Controller/LuckyController.php
@@ -11,13 +11,11 @@ use Symfony\Component\Routing\Annotation\Route;
 
 class LuckyController extends AbstractController
 {
-    private int $globalMinNumber;
     private $logger;
     private $luckyNumberGenerator;
 
-    public function __construct(int $globalMinNumber, $logger, $luckyNumberGenerator)
+    public function __construct($logger, $luckyNumberGenerator)
     {
-        $this->globalMinNumber = $globalMinNumber;
         $this->logger = $logger;
         $this->luckyNumberGenerator = $luckyNumberGenerator;
     }
@@ -27,7 +25,7 @@ class LuckyController extends AbstractController
      */
     public function number($max = 100): Response
     {
-        $number = $this->luckyNumberGenerator->getRandomNumber($max, $this->globalMinNumber);
+        $number = $this->luckyNumberGenerator->getRandomNumber($max);
 
         $this->logger->info('Generating a lucky number: {number}', [
             'number' => $number
diff --git a/src/Service/LuckyNumberGenerator.php b/src/Service/LuckyNumberGenerator.php
index d0a8e0f..3b99686 100644
--- a/src/Service/LuckyNumberGenerator.php
+++ b/src/Service/LuckyNumberGenerator.php
@@ -4,8 +4,15 @@ namespace App\Service;
 
 class LuckyNumberGenerator
 {
-    public function getRandomNumber(int $max, int $globalMinNumber): int
+    private int $globalMinNumber;
+
+    public function __construct(int $globalMinNumber)
+    {
+        $this->globalMinNumber = $globalMinNumber;
+    }
+
+    public function getRandomNumber(int $max): int
     {
-        return random_int($globalMinNumber, $max);
+        return random_int($this->globalMinNumber, $max);
     }
 }
