diff --git a/src/Controller/LuckyController.php b/src/Controller/LuckyController.php
index 806958f..c83af3f 100644
--- a/src/Controller/LuckyController.php
+++ b/src/Controller/LuckyController.php
@@ -2,6 +2,7 @@
 
 namespace App\Controller;
 
+use Psr\Log\LoggerInterface;
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\HttpFoundation\JsonResponse;
 use Symfony\Component\HttpFoundation\Response;
@@ -10,10 +11,12 @@ use Symfony\Component\Routing\Annotation\Route;
 class LuckyController extends AbstractController
 {
     private int $globalMinNumber;
+    private LoggerInterface $logger;
 
-    public function __construct(int $globalMinNumber)
+    public function __construct(int $globalMinNumber, LoggerInterface $logger)
     {
         $this->globalMinNumber = $globalMinNumber;
+        $this->logger = $logger;
     }
 
     /**
@@ -23,6 +26,10 @@ class LuckyController extends AbstractController
     {
         $number = random_int($this->globalMinNumber, $max);
 
+        $this->logger->info('Generating a lucky number: {number}', [
+            'number' => $number
+        ]);
+
         return $this->render('lucky/number.html.twig', [
             'number' => $number,
         ]);
