diff --git a/config/services.yaml b/config/services.yaml
index cab0f42..e5c518b 100644
--- a/config/services.yaml
+++ b/config/services.yaml
@@ -25,3 +25,6 @@ services:
         arguments:
             $globalMinNumber: 5
             $logger: '@logger'
+            $luckyNumberGenerator: '@App\Service\LuckyNumberGenerator'
+
+    App\Service\LuckyNumberGenerator:
diff --git a/src/Controller/LuckyController.php b/src/Controller/LuckyController.php
index 394a843..7e6d3b4 100644
--- a/src/Controller/LuckyController.php
+++ b/src/Controller/LuckyController.php
@@ -13,11 +13,13 @@ class LuckyController extends AbstractController
 {
     private int $globalMinNumber;
     private $logger;
+    private $luckyNumberGenerator;
 
-    public function __construct(int $globalMinNumber, $logger)
+    public function __construct(int $globalMinNumber, $logger, $luckyNumberGenerator)
     {
         $this->globalMinNumber = $globalMinNumber;
         $this->logger = $logger;
+        $this->luckyNumberGenerator = $luckyNumberGenerator;
     }
 
     /**
@@ -25,8 +27,7 @@ class LuckyController extends AbstractController
      */
     public function number($max = 100): Response
     {
-        $generator = new LuckyNumberGenerator();
-        $number = $generator->getRandomNumber($max, $this->globalMinNumber);
+        $number = $this->luckyNumberGenerator->getRandomNumber($max, $this->globalMinNumber);
 
         $this->logger->info('Generating a lucky number: {number}', [
             'number' => $number
