diff --git a/src/Controller/LuckyController.php b/src/Controller/LuckyController.php
index 9164d4d..9412b90 100644
--- a/src/Controller/LuckyController.php
+++ b/src/Controller/LuckyController.php
@@ -2,6 +2,7 @@
 
 namespace App\Controller;
 
+use App\Service\LuckyNumberGenerator;
 use Psr\Log\LoggerInterface;
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\HttpFoundation\Response;
@@ -23,7 +24,8 @@ class LuckyController extends AbstractController
      */
     public function number($max = 100): Response
     {
-        $number = random_int($this->globalMinNumber, $max);
+        $generator = new LuckyNumberGenerator();
+        $number = $generator->getRandomNumber($max, $this->globalMinNumber);
 
         $this->logger->info('Generating a lucky number: {number}', [
             'number' => $number
