diff --git a/src/Controller/LuckyController.php b/src/Controller/LuckyController.php
index 35128df..33ea873 100644
--- a/src/Controller/LuckyController.php
+++ b/src/Controller/LuckyController.php
@@ -23,6 +23,8 @@ class LuckyController extends AbstractController
      */
     public function number(LoggerInterface $logger, $max = 100): Response
     {
+        $this->denyAccessUnlessGranted('VIEW', $max);
+
         $number = $this->luckyNumberGenerator->getRandomNumber($max);
 
         $logger->info('Generating a lucky number: {number}', [
